packages/client/bin/cli.ts

```diff
@@ -300,9 +300,10 @@ async function run() {
         byzantiumBlock: 0,
         constantinopleBlock: 0,
         petersburgBlock: 0,
-        istanbulBlock: 0,
-        berlinBlock: 0,
-        londonBlock: 0,
+        istanbulBlock: 2,
+        berlinBlock: 4,
+        londonBlock: 6,
+        mergeBlock: 8,
```

Comment:
by the way geth defines this as `terminalTotalDifficulty` not `mergeBlock`, I have implemented it [here](https://github.com/ethereumjs/ethereumjs-monorepo/pull/1509/files#diff-972b7562b7b37e37b85f42447f901a9dcd65484f78d16e42144ff8cee2c11f88R254-R256)

Reply:
i also implemented [stopping the miner](https://github.com/ethereumjs/ethereumjs-monorepo/blob/master/packages/client/lib/miner/miner.ts#L73-L74), although I did it on `chainCommon` not `execCommon` - I believe `execCommon` should already work but I haven't implemented updating the `chainCommon` yet (will be doing that soon in my next work)